app/Config/App.php

```diff
@@ -199,6 +199,13 @@ class App extends BaseConfig
 	| 'cookiePath'     = Typically will be a forward slash
 	| 'cookieSecure'   = Cookie will only be set if a secure HTTPS connection exists.
 	| 'cookieHTTPOnly' = Cookie will only be accessible via HTTP(S) (no javascript)
+	| 'cookieSameSite' = Configure cookie SameSite setting. Defaults to '' to not set
```

Comment:
Per RFC, if no samesite attribute is given or it is invalid, it shall default to `None`. Should we do that too here?

Reply:
OK. I think I have misread things.

For the invalid samesite, Sec. 4.1.2.7 says `If the "SameSite" attribute's value is something other than these three known keywords, the attribute's value will be treated as "None".`

Reply:
Maybe setting CSRFSameSite also to '' would be a good idea? I don't know when browsers will start enforcing SameSite on cookies.

Reply:
Per my reading the RFC, `Lax` is the suggested value for CSRF.

Reply:
Yeah, so 'Lax' should be standard for CSRF it seems.